improver/cli/threshold.py

```diff
@@ -157,11 +163,15 @@ def process(
         raise ValueError("Cannot apply land-mask cube without in-vicinity processing")
 
     if collapse_coord == "realization":
-        # TODO change collapse_coord argument to boolean "collapse_realizations"
-        # (requires suite change)
+        each_threshold_func_list.append(collapse_realizations)
+    elif collapse_coord == "percentile":
+        cube = RebadgePercentilesAsRealizations()(cube)
         each_threshold_func_list.append(collapse_realizations)
     elif collapse_coord is not None:
-        raise ValueError("Cannot collapse over non-realization coordinate")
+        raise ValueError(
+            "Can only collapse over a realization coordinate or a percentile "
+            "coordinate that has been rebadged as a realization coordinate."
+        )
```

Comment:
Sorry, should have noticed before. Can you update the doc-string to indicate that only realizations or percentiles can be collapsed. No need for someone to hit the error to learn that.

Reply:
I've updated the docstring.